### DISCUSSION TITLE
how can I get apt command on chronos? I would like to download neovim

### DISCUSSION THREAD

**Discussion by 0nesnow** [dnschneid/crouton#5119] [Q&A]
I want to download neovim in chronos instead of in a crouton created chroot so I can use it in every one of of my chroots instead of downloading it in every one of my chroots individually
Is there a way to do it

**Comment by DennisLfromGA** [comment]
@0nesnow,

The apt package manager is not available on CrOS but you can download and use the neovim appimage, I use it as my default editor.

-**DennisLfromGA**

**Reply by 0nesnow** [reply]
Thanks is there anyway I can use application in all my chroots by just installing it in chronos
